src/agents/players/book_agent.py

```diff
@@ -0,0 +1,218 @@
+        [hit, stay, stay, stay, stay, stay, hit, hit, hit, hit],
+        [hit, stay, stay, stay, stay, stay, hit, hit, hit, hit],
+        [hit, stay, stay, stay, stay, stay, hit, hit, hit, hit],
+        [hit, stay, stay, stay, stay, stay, hit, hit, hit, hit],
+        [hit, stay, stay, stay, stay, stay, hit, hit, hit, hit],
+        [stay, stay, stay, stay, stay, stay, stay, stay, stay, stay],
+        [stay, stay, stay, stay, stay, stay, stay, stay, stay, stay],
+        [stay, stay, stay, stay, stay, stay, stay, stay, stay, stay],
+        [stay, stay, stay, stay, stay, stay, stay, stay, stay, stay],
+        [stay, stay, stay, stay, stay, stay, stay, stay, stay, stay]
+    ]
+
+    """
+    by the book action matrix for hands that aren't soft, can't split, and can't double
+        1  2  3  4  5  6  7  8  9  10
+    6   H  H  H  H  H  H  H  H  H  H
+    7   H  H  H  H  H  H  H  H  H  H
+    8   H  H  H  H  H  H  H  H  H  H
+    9   H  H  H  H  H  H  H  H  H  H
+    10  H  H  H  H  H  H  H  H  H  H
+    11  H  H  H  H  H  H  H  H  H  H
+    12  H  S  S  S  S  S  H  H  H  H
+    13  H  S  S  S  S  S  H  H  H  H
+    14  H  S  S  S  S  S  H  H  H  H
+    15  H  S  S  S  S  S  H  H  H  H
+    16  H  S  S  S  S  S  H  H  H  H
+    17  S  S  S  S  S  S  S  S  S  S
+    18  S  S  S  S  S  S  S  S  S  S
+    19  S  S  S  S  S  S  S  S  S  S
+    20  S  S  S  S  S  S  S  S  S  S
+    21  S  S  S  S  S  S  S  S  S  S
+    """
+    hard_actions_ndx_offset = 6
+    hard_actions = [
+        [hit, hit, hit, hit, hit, hit, hit, hit, hit, hit],
+        [hit, hit, hit, hit, hit, hit, hit, hit, hit, hit],
+        [hit, hit, hit, hit, hit, hit, hit, hit, hit, hit],
+        [hit, hit, hit, hit, hit, hit, hit, hit, hit, hit],
+        [hit, hit, hit, hit, hit, hit, hit, hit, hit, hit],
+        [hit, hit, hit, hit, hit, hit, hit, hit, hit, hit],
+        [hit, stay, stay, stay, stay, stay, hit, hit, hit, hit],
+        [hit, stay, stay, stay, stay, stay, hit, hit, hit, hit],
+        [hit, stay, stay, stay, stay, stay, hit, hit, hit, hit],
+        [hit, stay, stay, stay, stay, stay, hit, hit, hit, hit],
+        [hit, stay, stay, stay, stay, stay, hit, hit, hit, hit],
+        [stay, stay, stay, stay, stay, stay, stay, stay, stay, stay],
+        [stay, stay, stay, stay, stay, stay, stay, stay, stay, stay],
+        [stay, stay, stay, stay, stay, stay, stay, stay, stay, stay],
+        [stay, stay, stay, stay, stay, stay, stay, stay, stay, stay],
+        [stay, stay, stay, stay, stay, stay, stay, stay, stay, stay]
+    ]
+    
+    
+    """
+    by the book action matrix for hands that aren't soft but we can split
+        1  2  3  4  5  6  7  8  9  10  
+    4   H  H  H  H  H  H  H  H  H  H
+    6   H  H  H  H  H  H  H  H  H  H
+    8   H  H  H  H  H  H  H  H  H  H
+    10  H  D  D  D  D  H  H  H  H  H
+    12  H  H  H  H  H  H  H  H  H  H
+    14  P  P  P  P  P  P  P  P  P  P
+    16  P  P  P  P  P  P  P  P  P  P
+    18  P  P  P  P  P  P  P  P  P  P
```

Comment:
Some of the optimal strategies I've seen vary, but none on the first few results of Google images say to split 9's or 7's all the time. What guide are you using for optimal strategy?

Reply:
I didn't haha. I'll update this. Pretty much guessed at the strategy given what I remember. I'll fix it to include a real one I find online